src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo"
+          alt="moyo logo"
```

Comment:
The `alt` attribute should describe the image content. Try to be more specific about the image content. For example, 'Moyo company logo' instead of 'moyo logo'.